internal/k8shandler/reconciler.go

```diff
@@ -46,6 +46,9 @@ func Reconcile(requestCluster *logging.ClusterLogging, requestClient client.Clie
 			return fmt.Errorf("Unable to create or update logstore for %q: %v", clusterLoggingRequest.Cluster.Name, err)
 		}
 
+		if err = clusterLoggingRequest.CreateOrDeleteLogExplorationApi(); err != nil {
+			return fmt.Errorf("Unable to create or update logstore for %q: %v", clusterLoggingRequest.Cluster.Name, err)
```

Comment:
This should:
* log as an error but not exit reconciliation of ClusterLogging.  
* Update status to identify error issue with deployment of this service